latest-ocpnode-1208.txt

```diff
@@ -0,0 +1 @@
+4f29b3c7ca3ae250a4a782064b4151159bdc6e61
```

Comment:
Do we need that file?

Reply:
My bad, mistakenly added. Removed it now!